aleth/main.cpp

```diff
@@ -202,6 +202,9 @@ int main(int argc, char** argv)
     fs::path configPath;
     string configJSON;
 
+    ///List_logging_channels
+    const std::string listLogChannels = "block blockhdr bq chain client debug discov error ethcap exec host impolite info net peer snap sync tq trace vmtrace warn watch";
```

Comment:
```suggestion
    std::string const logChannels = "block blockhdr bq chain client debug discov error ethcap exec host impolite info net overlaydb peer rpc snap statedb sync timer tq trace vmtrace warn watch";
```
(some channels were also using `clog(...)` way to log, it was hard to find, sorry)

Also move this definition closer to where it's used (right before line 328)